rest-service/manager_rest/upload_manager.py

```diff
@@ -831,7 +844,7 @@ def _prepare_and_process_doc(self,
                                  archive_target_path,
                                  **kwargs):
         plugins = []
-        caravan_ = self.Caravan(archive_target_path)
+        caravan_ = kwargs['caravan_']
```

Comment:
this is a bit unclear, what will be returned here?

Reply:
Better?